Assets/Scripts/Human.cs

```diff
@@ -47,30 +48,32 @@ public class Human : Player {
         base.Update();
         // if game ended and trigger pressed, restart the game
         if (GameManager.Instance.isGameEnded) {
-            if (GameManager.Instance.isOculusQuest) {
-                if (OVRInput.GetDown(OVRInput.Button.PrimaryIndexTrigger) || OVRInput.GetDown(OVRInput.Button.SecondaryIndexTrigger)) {
+            if (GameManager.Instance.playerPlatform == PlayerPlatform.OCULUS) {
+                if (OVRInput.GetDown(OVRInput.Button.PrimaryIndexTrigger)
+                    || OVRInput.GetDown(OVRInput.Button.SecondaryIndexTrigger)) {
                     GameManager.Instance.Restart();
                 }
-            } else if (GameManager.Instance.isEditor) {
+            } else if (GameManager.Instance.playerPlatform == PlayerPlatform.EDITOR) {
                 if (Input.GetKeyDown(KeyCode.Space) || Input.GetKeyDown(KeyCode.Return)) {
                     GameManager.Instance.Restart();
                 }
 
-            } else if (trigger.GetStateDown(SteamVR_Input_Sources.Any)) {
-                GameManager.Instance.Restart();
+            } else if (GameManager.Instance.playerPlatform == PlayerPlatform.STEAMVR) {
+                if (trigger.GetStateDown(SteamVR_Input_Sources.Any)) {
+                    GameManager.Instance.Restart();
+                }
             }
         }
 
-        if (GameManager.Instance.isOculusQuest) {
+        if (GameManager.Instance.playerPlatform == PlayerPlatform.OCULUS) {
             if (OVRInput.GetDown(OVRInput.RawButton.X | OVRInput.RawButton.Y)) {
                 leftHandController.Switch();
             }
 
             if (OVRInput.GetDown(OVRInput.RawButton.A | OVRInput.RawButton.B)) {
                 rightHandController.Switch();
             }
-        } else if (!GameManager.Instance.isEditor) {
-            // if grab is pressed, switch between tool and spawn for that hand
+        } else if (GameManager.Instance.playerPlatform == PlayerPlatform.STEAMVR) {
```

Comment:
nope, `PlayerPlatform.STEAMVR` is right. The old one was a `!editor` because it was something like "if not oculus, and not editor, means it must be steam VR"

Reply:
ah ok sry missed the !